Samples/iOS-ObjectiveC/iOS-ObjectiveC/AppDelegate.m

```diff
@@ -57,7 +57,6 @@ - (BOOL)application:(UIApplication *)application
                 config.configureWidget = ^(SentryUserFeedbackWidgetConfiguration *_Nonnull widget) {
                     if ([args containsObject:@"--io.sentry.feedback.auto-inject-widget"]) {
                         widget.labelText = @"Report Jank";
-                        widget.widgetAccessibilityLabel = @"io.sentry.iOS-Swift.button.report-jank";
```

Comment:
`m`: Did I miss it, or do widgets not have any accessibility labels anymore?

Reply:
This actually is one of our[ configuration properties](https://github.com/getsentry/sentry-cocoa/blob/1d128a69ac159da380b8fa4c982f5e13f0f8a94b/Sources/Swift/Integrations/UserFeedback/Configuration/SentryUserFeedbackWidgetConfiguration.swift#L38) for the "user feedback widget", not for an [iOS widget](https://developer.apple.com/design/human-interface-guidelines/widgets), if that's what you mean.

Reply:
Are you thinking of iOS widgets? This is just a configuration option for our new feature, which sets an accessibility label for the floating button that is displayed in an app's UI.